### ISSUE TITLE
[FR] Set the on tap action for entities

### ISSUE THREAD

**Issue by jasonmhite** [dwainscheeren/dwains-lovelace-dashboard#490]
First off, great project! This saved me a ton of time finally getting around to redoing my dashboard after moving.

A feature that would be nice to have is the ability to set the action when tapping on an entity similar to how Lovelace button cards work. It'd be nice if lights/switches/etc could be made to toggle the corresponding entity without having to manually designate a Lovelace card.

**Comment by dwainscheeren** [comment]
If you click on the icon it toggles if you click on the name it opens the popup :)

**Comment by jasonmhite** [comment]
Well... oh. Duh.

Still though, it isn't very touch friendly. Is there a way to make it toggle on clicking anywhere on the tile and long press to see the popup?

**Comment by dwainscheeren** [comment]
In v1 and v2 there was always this discussion. Where to double tap, where not etc. That's why in v3 I made it this way. On icon is toggle on name/state is popup :)

**Comment by jasonmhite** [comment]
Fair enough, if that's how you prefer it! I'll leave it to you if you want to close this or leave it open and consider further.

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
